rest/taskrouter/activities/instance/delete/example-1/example-1.5.x.php

```diff
@@ -0,0 +1,19 @@
+<?php
+// Get the PHP helper library from twilio.com/docs/php/install
+require_once('/path/to/twilio-php/Services/Twilio.php'); // Loads the library
```

Comment:
As PHPCS will suggest (I think that the PHP interpreter will also give you a warning) `require_once` is not a function you should not use parenthesis. This is an error found in the v4 snippets. Already fixed in my v5 snippets.